process_config.sh

```diff
@@ -101,14 +78,10 @@ function find_dev_for_dir
     else
 	# get devices from partition
 	for dev in $(ls /sys/block/); do
-	    match=$(lsblk /dev/$dev | egrep "$PART|$REALPART")
-	    if [[ "$match" == "" ]]; then
-		continue
-	    fi
```

Comment:
Could you explain why this is not needed any longer? Be verbose, because I have learned these details while writing this side code, and I have forgotten virtually all of them :)